@types/express-session/index.d.ts

```diff
@@ -0,0 +1,8 @@
+export {};
+
+declare module "express-session" {
```

Comment:
Why not to use: https://www.npmjs.com/package/@types/express-session ?

Reply:
same as above, I'm just extending those declarations to use additional fields that admin saves to session